addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -56,7 +56,10 @@ options.registry.TableOfContent = options.Class.extend({
             $el.attr('id', id);
             $el[0].dataset.anchor = 'true';
         });
-        $nav.find('a:first').addClass('active');
+        const targetNode = this.$target.find('.s_table_of_content_main')[0]
```

Comment:
- Missing semicolon at the end of the line
- targetNode is a HTMLElement so the variable name should finish with `El`
- Maybe you can avoid adding new JQuery code (as we want to get rid of it) => `this.$target[0].querySelector(...`